pyodide-build/pyodide_build/config.py

```diff
@@ -89,9 +89,34 @@ def _load_config_from_env(self, env: Mapping[str, str]) -> Mapping[str, str]:
             BUILD_VAR_TO_KEY[key]: env[key] for key in env if key in BUILD_VAR_TO_KEY
         }
 
-    def _load_config_file(self) -> Mapping[str, str]:
-        # TODO: Implement this
-        return {}
+    def _load_config_file(
+        self, curdir: Path, env: Mapping[str, str]
+    ) -> Mapping[str, str]:
+        # avoid circular import
+        from pyodide_build.build_env import search_pyproject_toml
```

Comment:
Let's move `search_pyproject_toml` function to `common.py`, so we don't need to care about the circular import.